ci-operator/step-registry/openshift-qe/installer/bm/day2/lvms/openshift-qe-installer-bm-day2-lvms-commands.sh

```
@@ -33,8 +33,9 @@ wipe_nvme() {
   local bastion=$1
   ssh ${SSH_ARGS} root@${bastion} '
     for i in $(oc get node --no-headers -l node-role.kubernetes.io/worker --output custom-columns="NAME:.status.addresses[0].address"); do
+      ssh -t '"${INNER_SSH_ARGS}"' core@$i "sudo vgchange -an 2>/dev/null || true; sudo pvs --noheadings -o vg_name 2>/dev/null | awk '{print \$1}' | sort -u | xargs -r -I{} sh -c 'echo y | sudo vgremove -ff {} 2>/dev/null || true'" 2>/dev/null
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)